ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml

```diff
@@ -394,6 +394,25 @@ tests:
     - ref: cucushift-helper-service-build-info
     - ref: openshift-extended-test
     workflow: rosa-aws-sts-hypershift
+- as: aws-rosa-sts-hypershift-prod-guest-p1-f7
```

Comment:
@YuLi517 @jhou1 Will hypershift need to follow the test strategy for nightly/stable payloads we did recently ?

Reply:
This is for rosa-hypershift. rosa env can only use stable image to run the test.

Reply:
Ack